src/runner.rb

```diff
@@ -17,6 +18,23 @@ class NotifyOrganizationRetryableError < StandardError; end
   RUNNER_INPUT = {}.freeze
 end
 
-RUNNER_CODE = RUNNER_INPUT.fetch('code', nil) || JSON.parse(ENV.fetch('RUNNER_CODE', '{}'))
-FIELDS = RUNNER_INPUT.fetch('fields', nil) || JSON.parse(ENV.fetch('RUNNER_FIELDS', '{}'))
-ACCOUNT = RUNNER_INPUT.fetch('account', nil) || JSON.parse(ENV.fetch('RUNNER_ACCOUNT', '{}'))
+def parse_input(runner_input, environment, key)
+  caps_key = "RUNNER_#{key.upcase}"
+  base64_key = "#{caps_key}_BASE64"
+
+  value = runner_input.fetch(key, nil)
+  return value unless value.nil?
+
+  value = environment.fetch(caps_key, nil)
+  return JSON.parse(value) unless value.nil?
+
+  value = environment.fetch(base64_key, nil)
+  return {} if value.nil?
+
+  decoded_value = Base64.decode64(value)
+  decoded_value.to_s == '' ? {} : JSON.parse(decoded_value)
```

Comment:
Vi räknar alltid med att avkodad base64-data är JSON här?

Reply:
Japp, är ju då från denna `'RUNNER_CODE_BASE64' => '{{ input.body | json | base64_encode }}',`

Reply:
Inget alls eller JSON med Base64.

Reply:
Såg den andra PR:en nu, blev mer logiskt då :) 

Gäller det här problemen med t.ex. Kemlab? Varför hjälper det att konvertera fram och tillbaka mellan base64?

Reply:
Kan ju vara värt att komma ihåg om vi liknande problem på andra steg där vi kör samma sätta att skicka in data sen.

Reply:
Intressant! 👍 Approvar dem